metro.config.js

```diff
@@ -2,33 +2,41 @@ const path = require( 'path' );
 const fs = require( 'fs' );
 const metroResolver = require( 'metro-resolver' );
 
+const nodeModulePaths = [
+	'../../node_modules',
+	'../../../jetpack/projects/plugins/jetpack/node_modules',
+];
+
 const gutenbergMetroConfig = require( './gutenberg/packages/react-native-editor/metro.config.js' );
 const extraNodeModules = {};
 const gutenbergMetroConfigCopy = {
 	...gutenbergMetroConfig,
+	projectRoot: path.resolve( __dirname ),
 	resolver: {
 		...gutenbergMetroConfig.resolver,
+		unstable_enableSymlinks: false,
```

Comment:
I think we may be able to keep this enabled. I may follow up with a refactor later.